packages/synthetic-chain/src/lib/vat-status.js

```diff
@@ -75,25 +77,36 @@ const makeSwingstore = db => {
   });
 };
 
-/**
- * @param {string} vatName
- */
-export const getVatDetails = async vatName => {
+function makeSwingstore() {
   const fullPath = swingstorePath.replace(/^~/, NonNullish(HOME));
-  const kStore = makeSwingstore(dbOpenAmbient(fullPath, { readonly: true }));
+  return makeSwingstoreTool(dbOpenAmbient(fullPath, { readonly: true }));
+}
 
-  const vatID = kStore.findVat(vatName);
+/** @typedef {ReturnType<typeof makeSwingstore>} Swingstore
```

Comment:
`makeSwingstore` is just a local helper to make what `makeSwingstoreTool` returns, which isn't a "Swingstore".

It's good to have a type for the `kStore` param but instead it should be,
```suggestion
/** @typedef {ReturnType<typeof makeSwingstoreTool>} SwingstoreTool
```
and go below the `makeSwingstoreTool` declaration.

Once that's done this is good to publish so go ahead and bump the version number. Then I'll approve and publish it and you can merge to main.

Reply:
done